library/test/Cargo.toml

```diff
@@ -17,6 +17,8 @@ panic_abort = { path = "../panic_abort" }
 
 # not actually used but needed to always have proc_macro in the sysroot
 proc_macro = { path = "../proc_macro" }
+# not actually used but needed to always have profiler_builtins in the sysroot
+profiler_builtins = { path = "../profiler_builtins" }
```

Comment:
I'd prefer to do this explicitly in bootstrap rather than hacking it in here

Reply:
If `profiler_builtins` is not present in the dependencies, i guess it won't be not compiled?

Reply:
Oh, I just saw this is matching what we do for `proc_macro`. Not super happy about that either but not something you need to fix in this PR, it's fine as-is.